src/display/annotation_layer.js

```diff
@@ -573,16 +573,22 @@ class AnnotationElement {
     return fields;
   }
 
-  show() {
+  show(restorePreviousePopup = true) {
```

Comment:
This parameter *appears* completely unused, since I cannot find a call-site setting it, why do we need it?
Also, having an optional parameter where `undefined` implies `true` might generally not be the most logical?
(Note that the spelling is slightly wrong: `restorePreviousPopup`.)